CHANGELOG.md

```diff
@@ -37,6 +37,7 @@
 - Remove unnecessary mock json files for frontend E2E tests by [@dawoodkhan82](https://github.com/dawoodkhan82) in [PR 4625](https://github.com/gradio-app/gradio/pull/4625)
 - Update depedencies by [@pngwn](https://github.com/pngwn) in [PR 4643](https://github.com/gradio-app/gradio/pull/4643)
 - Remove `cleared_value` from some components as its no longer used internally by [@freddyaboulton](https://github.com/freddyaboulton) in [PR 4685](https://github.com/gradio-app/gradio/pull/4685)
+- Fix the `description` of the `@gradio/client` package by [@whitphx](https://github.com/whitphx) in [PR 4717](https://github.com/gradio-app/gradio/pull/4717)
```

Comment:
We don't need a changelog entry for this but we do need a changeset (i know it is confusing).

We use `changesets` to handle versioning the js stuff.  If you run `pnpm changeset add` you will get an interactive CLI. Choose the client library and a patch change and you can enter a changelog entry. Then commit this file with everything else.

If you want me to take care of this then I am happy to!

Reply:
Got it!
I added the edits.